### PR TITLE
use LoadingInfo for ExpertsView

### PR DESCRIPTION
**Pull request by DelFlumen**
develop

## GitHub Board

**Issue link**

[#51]( )

## Summary of change

use LoadingInfo for ExpertsView

### CODE REVIEW

File: src/lib/components/ExpertsView.tsx [hunk 1]
```diff
@@ -48,18 +45,13 @@ export const ExpertsView: React.FC<IExpertsViewProps> = (props) => {
...
 
   return (
     <Container>
       <Typography variant="h4">Експерти</Typography>
       <Grid container direction="column" alignItems="center">
-        {renderLoading()}
-        {renderError()}
+        {loading && <LoadingInfo loading={loading} errorMsg={errorMsg} />}
       </Grid>
       <div className={classes.container}>
```
Comment (SergeOlabin) [inline src/lib/components/ExpertsView.tsx:56]: i guess this should not be shown when `loading`

**Review by SergeOlabin — APPROVED** [review]

File: src/lib/components/ExpertsView.tsx (hunk 1 above)
Comment (DelFlumen) [reply src/lib/components/ExpertsView.tsx:56]: It's strange, but I don't have these 61 & 62 lines in my code.  I've pushed again.

Comment (SergeOlabin) [reply src/lib/components/ExpertsView.tsx:56]: no it's okay :)
i mean, not to show all experts container on line:56 when `loading`

**Review by SergeOlabin — CHANGES_REQUESTED** [review]

File: src/lib/components/ExpertsView.tsx (hunk 1 above)
Comment (DelFlumen) [reply src/lib/components/ExpertsView.tsx:60]: fixed

**Review by SergeOlabin — APPROVED** [review]

**Pull request merged**